WcaOnRails/config/locales/en.yml

```diff
@@ -1459,6 +1459,12 @@ en:
         search: "Name or city"
         #context: the "recent" button
         recent: "Last %{count} days"
+        #context: the registration status icon
+        registration:
+          opens_in: "Registration will open in %{days}"
+          closed: "Registration has closed"
+          open: "Registraion is open!"
+          full: "Competitor limit reached, the waiting list is now in effect"
```

Comment:
@Jambrose777 Do we have already have some standardized wording for "the waiting list is now in effect" from the Competition Announcement Team?

Reply:
Not Jacob but I do think "waiting list in effect" is not a commonly used term. What about "new registrations will go to the waiting list"?